spark-jobs/src/test/resources/application.conf

```diff
@@ -13,7 +13,7 @@
 
 enceladus.version=${project.version}
 
-menas.rest.uri="http://localhost:8080/menas/api"
+enceladus.rest.uri="http://localhost:8080/menas/api"
```

Comment:
This seems to be always overwritten in practice with actual URI when running, but providing a different sensible default/hint (e.g. `http://localhost:8080/rest_api`) would make more sense to me.